### PR TITLE
fix: lighthouse complaining about robots.txt

### PR DESCRIPTION
**Pull request by tlgimenes**
## What's the purpose of this pull request?
Fix lighthouse complaining about robots.txt.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -58,6 +58,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - A missing gap between the Sign In link and Cart button on desktop ([#11](https://github.com/vtex-sites/nextjs.store/pull/11)).
 - A bugged vertical gap with the `EmptyState` component inside the cart ([#11](https://github.com/vtex-sites/nextjs.store/pull/11)).
 - Fixes Storybook build removing unused imports ([#40](https://github.com/vtex-sites/nextjs.store/pull/40))
+- Lighthouse complaining about missing robots.txt
```
Comment (hellofanny) [inline CHANGELOG.md:61]:
```suggestion
- Lighthouse complaining about missing robots.txt ([#89](https://github.com/vtex-sites/nextjs.store/pull/89))
```

*And the latest/newest entries should go on top. 👍

**Review by hellofanny — APPROVED** [review]
LGTM! Just remember to update the changelog, ok? 🙂

**Review by filipewl — APPROVED** [review]
Fixes this:
CleanShot 2022-06-02 at 12 59 33

👏 

After @hellofanny's suggestion, this looks good to me as well!

File: CHANGELOG.md (hunk 1 above)
Comment (filipewl) [reply CHANGELOG.md:61]: I understand it's a pain to have to send an extra commit only after the PR is opened so we know its number.
🫂

**Pull request merged**